### ISSUE TITLE
Some small typos in chapter 5

### ISSUE THREAD

**Issue by simoncarlen** [uu-sml/sml-book-page#52]
**Location**
Please provide:
1. April 30, 2021
2. page 81, 82, 84
3. p. 81 (a): the sentence just above the box explaining the training objective
p. 81 (b): under section 'Optimization accuracy vs. statistical accuracy'
p. 82 (a): line 4
p. 82 (b): line 6-7
p. 84: third line from bottom

**Describe the mistake**
p. 81 (a): "However, this insight is still of practical importance, because ***is*** means that"
p. 81 (b): "In practice it can difficult to determine"
p. 82 (a): "However, due the noise in the data"
p. 82 (b): "with the aim of making the optimization problem (5.4) ***is*** easier to solve"
p. 84: "While cross entropy is commonly used in practice, ***but*** there are also other loss functions for binary classification that are useful."

**Suggested change**
p. 81 (a): is means that --> it means that
p. 81 (b) In practice it can ***be*** difficult to determine
p. 82 (a) However, due ***to*** the noise in the data
p. 82 (b): Remove "is" --> with the aim of making the optimization problem (5.4) easier to solve
p. 84: Perhaps remove "but"

**Comment by andreas-lindholm** [comment]
Thank you
